openwisp_controller/config/base/template.py

```diff
@@ -61,7 +67,7 @@ class AbstractTemplate(ShareableOrgMixinUniqueName, BaseConfig):
         default=False,
         db_index=True,
         help_text=_(
-            "whether new configurations will have this template enabled by default"
+            "whether all configurations will have this template enabled by default"
```

Comment:
I would like to take advantage to update this label. 

What do you think of the following?

```
whether this template is applied to all current and future devices by default (can be unassigned manually)
```